src/otx/recipe/zero_shot_visual_prompting/openvino_model.yaml

```diff
@@ -6,7 +6,8 @@ model:
     model_type: Zero_Shot_Visual_Prompting
     async_inference: False
     use_throughput_mode: True
-    root_reference_info: vpm_zsl_reference_infos
+    reference_info_dir: reference_infos
+    infer_reference_info_root: ../.latest/train # set absolute path for using reference_info saved in other location
```

Comment:
If the same value is already declared as the default value for the class, is there any other reason to add it per recipe? It seems like it could be rather confusing.

Reply:
Actually to let users know to set these arguments. I thought it would be better to let users know how to update these arguments without having to check them code-based.
Do you have any other alternatives? If so, please recommend :)